README.md

```diff
@@ -1,39 +1,58 @@
-# MCP(Model Context Protocol) for Dart
+# MCP (Model Context Protocol) for Dart
 
-[![Pub Version](https://img.shields.io/pub/v/mcp_dart?color=blueviolet)](https://pub.dev/packages/mcp_dart)
+[![Pub
+Version](https://img.shields.io/pub/v/mcp_dart?color=blueviolet)](https://pub.dev/packages/mcp_dart)
 [![likes](https://img.shields.io/pub/likes/mcp_dart?logo=dart)](https://pub.dev/packages/mcp_dart/score)
 
-[Model Context Protocol](https://modelcontextprotocol.io/) (MCP) is an open protocol designed to enable seamless integration between LLM applications and external data sources and tools.
+[Model Context Protocol](https://modelcontextprotocol.io/) (MCP) is an open
+protocol designed to enable seamless integration between LLM applications and
+external data sources and tools.
 
-This library aims to provide a simple and intuitive way to implement MCP servers and clients in Dart, while adhering to the [MCP protocol spec](https://spec.modelcontextprotocol.io/). The goal is to make this SDK as similar as possible to the official SDKs available in other languages, ensuring a consistent developer experience across platforms.
+This library provides a simple and intuitive way to implement MCP servers and
+clients in Dart. **MCP clients now support all platforms including web browsers
+and WASM**, while MCP servers remain optimized for Dart VM environments. It
+adheres to the [MCP protocol spec](https://spec.modelcontextprotocol.io/) while
+ensuring a consistent developer experience across all platforms.
 
 ## Requirements
 
 - Dart SDK version ^3.0.0 or higher
 
-Ensure you have the correct Dart SDK version installed. See <https://dart.dev/get-dart> for installation instructions.
+Ensure you have the correct Dart SDK version installed. See
+<https://dart.dev/get-dart> for installation instructions.
 
 ## Features
 
-- Stdio support (Server and Client)
-- StreamableHTTP support (Server and Client)
-- SSE support (Server only) - Deprecated
-- Stream Transport using dart streams (Server and Client in shared process)
-- Tools
-- Resources
-- Prompts
-- Sampling
-- Roots
+- **Cross-Platform Client Support** - MCP clients work on all platforms: Dart
+  VM, Flutter (mobile/desktop), web browsers, and WASM
+- **Conditional Compilation** - Platform-optimized implementations with
+  identical APIs
+- **MCP Servers** (Dart VM only):
+  - Stdio support (Server and Client)
+  - StreamableHTTP support (Server only)
+  - SSE support (Server only) - Deprecated  
+  - Stream Transport using dart streams (Server and Client in shared process)
+- **MCP Clients** (Cross-platform):
+  - Stdio support (VM only)
+  - **StreamableHTTP support** - **Works everywhere: VM, web browsers, WASM!**
```

Comment:
I feel it's redundant. Maybe we don't need to split Client/Server in this section.